src/main/kotlin/Utilities/FileInterface.kt

```diff
@@ -1,7 +1,10 @@
 package Utilities
 
+import java.io.BufferedWriter
+
 interface FileInterface {
     fun readResource(filePath: String): String
     fun writeResource(filePath: String, resource: String)
     fun readFile(fileName: String): String
+    fun clearFile(filePath: String)
```

Comment:
I meant to say this in another PR, but I think the naming is a bit confusing here. I think it would help to think about when to use `File` vs. `Resource`. 

Right now, you are using `File` when accessing the `resources` directory and `Resource` when accessing a file outside the `resources` directory. Maybe the names could have been flipped? I'm not sure it's worth doing at this point.

What do you think about `clearFile`'s name?